README.md

```diff
@@ -117,7 +117,7 @@ Or directly (without additional options) [only on Linux]:
 
     clang -g source.c -Xclang -load -Xclang ./sl_build/libsl.so
 
-Dependencies for Darwin (MacOSX)
+Dependencies for Darwin (macOS)
 ================================
```

Comment:
The horizontal line could be truncated by one `=`.

Reply:
Addressed in 2267963.